### PR TITLE
Development

### PR DESCRIPTION
**Pull request by salac-kolosek**

### CODE REVIEW

File: app/assets/stylesheets/constants/_colors.scss [hunk 1]
```diff
@@ -1,9 +1,10 @@
-$color1: #a9d142; /* green variant */
-$color2: #f6fbff; /* bluish gray */
-$color3: #eaeaea; /* gray variant */
-$color4: #687e87; /* light gray */
-$color5: #29a0e0; /* blue variant */
-$color6: #f2f8fA; /* light gray variant */ 
-$color7: #ffffff; /* white */
-$color8: #6dc4f3; /* dark blue */
-$color9: #8c8c8c; /* darker gray */
+$color1: #a9d142; //* green variant *//
```
Comment (kolosek) [inline app/assets/stylesheets/constants/_colors.scss]: It's sufficient to add only //   * and ending *// is not necessary ;) - FAQ

File: app/mailers/invitation_mailer.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class InvitationMailer < ApplicationMailer
+  default from: "ghost_blog_sale@kolosek.com"
```
Comment (kolosek) [inline app/mailers/invitation_mailer.rb]: Write this in configuration file, so this kind of things can be changed easily - write in FAQ, with example.

File: app/models/profile.rb [hunk 3]
```diff
@@ -2,4 +2,14 @@ class Profile < ApplicationRecord
...
   validates :full_name, presence: true
+
+  mount_uploader :avatar, AvatarUploader
+
+  before_destroy :softly_remove_user
+
+  private
+
+  def softly_remove_user
+    user.update(status: User::REMOVED)
```
Comment (kolosek) [inline app/models/profile.rb]: write a method for updating user removed status, and test for it.  - FAQ

File: spec/features/profiles_spec.rb [hunk 4]
```diff
@@ -1,13 +1,13 @@
 require "rails_helper"
 
 describe "Profiles" do
-  before :all do
-    @user = create(:user_with_profile_and_role)
+  before :each do
+    @user = create(:user_with_profile_and_admin_role)
```
Comment (kolosek) [inline spec/features/profiles_spec.rb]: You can just create factory called :admin

File: app/assets/stylesheets/constants/_colors.scss (hunk 1 above)
Comment (salac-kolosek) [reply app/assets/stylesheets/constants/_colors.scss]: Will be fixed.

File: app/models/profile.rb (hunk 3 above)
Comment (salac-kolosek) [reply app/models/profile.rb]: Ok.

**Pull request merged**